services/request_service.go

```diff
@@ -61,6 +94,23 @@ func (rs RequestService) ResolveDIDDocVersion(c echo.Context) error {
 
 	version := c.Param("version")
 
+	didMethod, _, identifier, _ := didUtils.TrySplitDID(did)
```

Comment:
As a part of future refactoring let's get rid of this copy-paste

Reply:
Yes, sure! Make sense!